bot/exts/evergreen/movie.py

```diff
@@ -89,7 +89,7 @@ async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
 
         # Get movies list from TMDB, check if results key in result. When not, raise error.
         movies = await self.get_movies_list(self.http_session, MovieGenres[genre].value, page)
-        if 'results' not in movies.keys():
+        if "results" not in movies.keys():
```

Comment:
The `.keys` here is unnecessary

Reply:
Also I can't comment there, but the `get_movies_list` function is misleadingly named imo, when it returns a `dict`; unimportant though so you can leave it alone if you don't wanna change it

Reply:
It does return a list, so I'll just change the return annotation to `List[Dict[str, Any]]`.